RocketChat/Rocket.Chat
# [FIX] Replace encrypted text to Encrypted Message Placeholder

## Proposed changes (including videos or screenshots)


### before 
![image](https://user-images.githubusercontent.com/27704687/150807900-154a9cdb-ee13-4333-8628-f287ab914b40.png)

### after
<img width="814" alt="Screenshot 2022-01-13 at 8 57 47 PM" src="https://user-images.githubusercontent.com/58601732/149359411-23e2430b-89e4-48b4-a3ad-65471d058551.png">



## Issue(s)


## Steps to test or reproduce

1. Setup an e2ee channel with 2 users.
2. Let one user save the e2e keys and configure e2ee and another user should not set up e2ee.
3. Placeholder should be shown for messages which cannot be decrypted.
## Further comments
